src/manifestoo/addons_path.py

```diff
@@ -54,11 +54,17 @@ def extend_from_import_odoo(self, python: str) -> None:
             output = tempfile.NamedTemporaryFile(delete=False)
             try:
                 output.close()
+                stderrpipe = subprocess.DEVNULL
+                stdoutpipe = subprocess.DEVNULL
+                if echo.verbosity > 0:
+                    stderrpipe = None
+                if echo.verbosity > 1:
+                    stdoutpipe = None
```

Comment:
@yann-papouin Thanks for this. Makes sense overall.
Do you have a specific reason to treat stdout and stderr differently here?

Reply:
No specific reason :)